### PR TITLE
Link to backend CA

### PR DESCRIPTION
**Pull request by leftclickben**
RWAHS-523
Link from detail page to CA backend

Also adds a couple of missing tests.

### CODE REVIEW

File: public/app/config/env/development.js [hunk 1]
```diff
@@ -29,6 +29,11 @@
 
             return function () {
                 container.register('options.service', options);
+                container.register('options.providence', {
+                    baseUrl: {
+                        objects: 'https://staging-collections.histwest.org.au/index.php/editor/objects/ObjectEditor/Summary/'
```
Comment (leftclickben) [inline public/app/config/env/development.js]: This matches the service definition options further up in the file; development frontend -> staging CA

File: public/app/ui/pages/detail/DetailPage.js [hunk 2]
```diff
@@ -37,6 +38,10 @@
                     return 'collections/' + context.params.type + '/detail';
                 });
 
+                this.curatorUrl = ko.pureComputed(function () {
+                    return providence.baseUrl.objects + 'object_id/' + context.params.id;
```
Comment (leftclickben) [inline public/app/ui/pages/detail/DetailPage.js]: The `object_id` is part of a key/value pair, zend framework style, so I kept it here.

File: test/spec/config/env/development.spec.js [hunk 3]
```diff
@@ -18,6 +18,10 @@
                     beforeEach(function () {
                         configure();
                     });
+                    it('Registers options', function () {
+                        expect(container.isRegistered('options.service')).to.equal(true);
```
Comment (leftclickben) [inline test/spec/config/env/development.spec.js]: This was being registered before, but the test was missing.

File: test/spec/config/env/development.spec.js [hunk 4]
```diff
@@ -18,6 +18,10 @@
...
+                        expect(container.isRegistered('options.providence')).to.equal(true);
```
Comment (leftclickben) [inline test/spec/config/env/development.spec.js]: This is for the new one.

File: test/spec/config/env/development.spec.js [hunk 5]
```diff
@@ -31,6 +35,9 @@
                         expect(container.isRegistered('detail.museum')).to.equal(true);
                         expect(container.isRegistered('detail.memorials')).to.equal(true);
                     });
+                    it('Registers the overlay utility', function () {
+                        expect(container.isRegistered('ui.overlay')).to.equal(true);
```
Comment (leftclickben) [inline test/spec/config/env/development.spec.js]: This was also registered, but not tested.

File: public/app/ui/pages/detail/DetailPage.js (hunk 2 above)
Comment (kehh) [inline public/app/ui/pages/detail/DetailPage.js]: makes sense

**Pull request merged**